keepassxc-browser/_locales/en/messages.json

```diff
@@ -0,0 +1,742 @@
+    },
+    "defineChoosePassword": {
+        "message": "2. Now choose a password field",
+        "description": "Choosing a password field text when choosing custom credential fields."
+    },
+    "defineConfirmSelection": {
+        "message": "3. Confirm selection",
+        "description": "Confirm a selection text when choosing custom credential fields."
+    },
+    "defineHelpText": {
+        "message": "Please confirm your selection or choose more fields as <em>String fields</em>.",
+        "description": "Confirm a selection text when choosing custom credential fields which contains string fields."
+    },
+    "username": {
+        "message": "Username",
+        "description": "General text for username."
+    },
+    "password": {
+        "message": "Password",
+        "description": "General text for password."
+    },
+    "credentialsNoUsername": {
+        "message": "- no username -",
+        "description": "Shown when no username is set in the credentials."
+    },
+    "credentialsNoLoginsFound": {
+        "message": "Error:\nNo logins found.",
+        "description": "Shown when no credentials are found for the current page."
+    },
+    "credentialsMultipleFound": {
+        "message": "Error:\nMore than one login was found in KeePassXC!\nPress the KeePassXC-Browser icon for more options.",
+        "description": "Alert message when trying to fill username and/or password when multiple credentials are found."
+    },
+    "credentialsNoUsernameFound": {
+        "message": "Error:\nNo credentials for given username found.",
+        "description": "Alert message when no credentials are found for the given username."
+    },
+    "fieldsFill": {
+        "message": "Error:\nCannot find fields to fill in.",
+        "description": "Alert message when no fields are found to fill in."
+    },
+    "fieldsNoPasswordField": {
+        "message": "Error:\nUnable to find a password field.",
+        "description": "Message shown when no password fields are found."
+    },
+    "rememberNothingChanged": {
+        "message": "Error:\nCould not detect changed credentials.",
+        "description": "Message shown when trying to save credentials that haven't changed."
+    },
+    "popupTitle": {
+        "message": "KeePassXC - Popup",
+        "description": "Popup window title."
+    },
+    "popupSettingsText": {
+        "message": " Settings",
+        "description": "Popup Settings button text."
+    },
+    "popupChooseCredentialsText": {
+        "message": " Choose custom credential fields for this page",
+        "description": "Popup credential choosing button text."
+    },
+    "popupConnectButton": {
+        "message": " Connect",
+        "description": "Popup Connect button text."
+    },
+    "popupReconnectButton": {
+        "message": " Reconnect",
+        "description": "Popup Reconnect button text."
+    },
+    "popupRedetectButton": {
+        "message": " Redetect credential fields",
```

Comment:
Maybe it's about time to change this to "login fields". Less jargon is always a good idea.